dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLiteConnector.java

```diff
@@ -50,72 +55,54 @@ public SnowflakeLiteConnector() {
 
   @Override
   @Nonnull
-  public String getDefaultFileName(boolean isAssessment, Clock clock) {
+  public String getDefaultFileName(boolean isAssessment, @Nullable Clock clock) {
     return ArchiveNameUtil.getFileName(NAME);
   }
 
-  private void addSqlTasksWithInfoSchemaFallback(
-      @Nonnull List<? super Task<?>> out,
-      @Nonnull Class<? extends Enum<?>> header,
-      @Nonnull String format,
-      @Nonnull String schemaZip,
-      @Nonnull String alteredSchemaView,
-      @Nonnull String usageZip,
-      @Nonnull String usageView,
-      @Nonnull String usageFilter) {
+  private ImmutableList<Task<?>> sqlTasksWithInfoSchemaFallback(
+      Class<? extends Enum<?>> header, String format, String schemaZip, String usageZip) {
+    String usageView = "SNOWFLAKE.ACCOUNT_USAGE";
+    String usageFilter = " WHERE DELETED IS NULL";
```

Comment:
nit: these 3 Strings could be marked as `final`

Reply:
For final attributes I agree. For final variables I think we should decide to either use them whenever possible or to never do it. Using final variables from time to time decreases readability - it's hard to tell if a missing `final` means the variable will be modified or the keyword was simply skipped in this case.

Current state of the Dumper seems closer to never using final variables than to always using them, so I'm following this trend.